webapp/portlet/src/main/webapp/share-activity-drawer/components/ShareActivityDrawer.vue

```diff
@@ -54,6 +58,13 @@ export default {
     shareDisabled() {
       return !this.spaces || this.spaces.filter(part => part !== '').length === 0;
     },
+    checkUrl() {
+      if (this.description) {
```

Comment:
```suggestion
      return this.description && this.description.match(REGEX);
```
the description should contain at least one link having one the below pattern:
https://www.XXXX.YYYY
http://www.XXXX.YYYY
www.XXXX.YYYY
https://XXXX.YYYY
http://XXXX.YYYY